ample/ensembler/truncation_util.py

```diff
@@ -400,67 +303,60 @@ def calculate_truncations(self,
         # No THESEUS variances required if scores for each residue provided
         var_by_res = run_theseus.var_by_res if truncation_method != "scores" \
             else self._convert_residue_scores(residue_scores)
-            
         if not len(var_by_res) > 0:
```

Comment:
Maybe better to use `if len(var_by_res) <= 0` ...

Reply:
Definitely!